pkg/env/env.go

```diff
@@ -143,18 +146,46 @@ func (v *ViperEnv) setupConfigFile() error {
 		}
 		return fmt.Errorf("could not read configuration cfg: %s", err)
 	}
+
+	v.Viper.WatchConfig()
+
+	dynamicLogHandler := func(env Environment) func(event fsnotify.Event) {
```

Comment:
this would be better to be in the log package, as this package is not be interested in log config changes

Reply:
normally i would put it there but then the proxies will have to manually add this handler also and I'd prefer for it to be transparent (always part of the env)